azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -50,6 +51,10 @@ internal class CallCompositeActivity : AppCompatActivity() {
     private val instanceId get() = intent.getIntExtra(KEY_INSTANCE_ID, -1)
 
     override fun onDestroy() {
+        if (!CallCompositeConfiguration.hasConfig(instanceId)) {
```

Comment:
Leave a reasoning here. It's a small enough block without enough context that the "why" will get easily lost.

e.g. Covers edge case where Android tries to recreate call activity after process death (e.g. due to revoked permission). If no configs are detected we can just exit without cleanup.

Reply:
Resolved with 21f4ef3